docs/src/components/home/Testimonials.tsx

```diff
@@ -0,0 +1,183 @@
+import * as React from 'react';
+import SwipeableViews from 'react-swipeable-views';
+import Avatar from '@material-ui/core/Avatar';
+import Box from '@material-ui/core/Box';
+import Container from '@material-ui/core/Container';
+import Grid from '@material-ui/core/Grid';
+import IconButton from '@material-ui/core/IconButton';
+import Typography from '@material-ui/core/Typography';
+import KeyboardArrowLeftRounded from '@material-ui/icons/KeyboardArrowLeftRounded';
+import KeyboardArrowRightRounded from '@material-ui/icons/KeyboardArrowRightRounded';
+import SvgUnity from '../../icons/SvgUnity';
+
+const data = [
+  { title: '2M', metadata: 'Weekly downloads on npm' },
+  { title: '69k', metadata: 'Stars on GitHub' },
+  { title: '2k', metadata: 'Open-source contributors' },
+  { title: '13k', metadata: 'Followers on Twitter' },
+];
+
+const UserFeedback = ({
+  quote,
+  profile,
+}: {
+  quote: string;
+  profile: { avatar: string; name: string; role: string; company?: React.ReactElement };
+}) => {
+  return (
+    <Box>
+      <Typography variant="subtitle1" color="#fff" sx={{ mb: 2 }}>
+        {quote}
+      </Typography>
+      <Box sx={{ display: 'flex' }}>
+        <Avatar
+          src={profile.avatar}
+          alt={`Picture of ${profile.name}`}
+          sx={{ width: 60, height: 60, border: '1px solid', borderColor: '#fff' }}
+        />
+        <Box sx={{ ml: 2 }}>
+          <Typography color="#fff" fontWeight="bold" sx={{ mb: 1 }}>
+            {profile.name},{' '}
+            <Box component="span" sx={{ color: 'grey.500', fontWeight: 'regular' }}>
+              {profile.role}
+            </Box>
+          </Typography>
+          {profile.company}
+        </Box>
+      </Box>
+    </Box>
+  );
+};
+
+const TESTIMONIALS = [
```

Comment:
I just added to Figma the company logos from the other two testimonials we have up to now. [Here it is.](https://www.figma.com/file/4uv2dT18rXJPZBbrbpw61q/Marketing-site?node-id=3311%3A15332)